BackofficeBundle/Resources/public/coffee/orchestraLib.coffee

```diff
@@ -164,13 +164,7 @@ activateTinyMce = (view, textarea) ->
       textarea.each ->
         if $(this).data('required')
           $(this).attr('required', 'required')
-      textarea.css(
-         display: 'block'
-         position: 'absolute'
-         width: '1px'
-         height: '1px'
-         top: '0px'
-         zIndex: '1')
+      textarea.css({display: 'block'}).addClass('focusable')
```

Comment:
Can we remove css diplay from js to put it in css files using a new class instead ?